bbot/test/test_step_1/test_manager_scope_accuracy.py

```diff
@@ -340,6 +340,9 @@ def custom_setup(scan):
     assert 1 == len([e for e in events if e.type == "IP_ADDRESS" and e.data == "127.0.0.77" and e.internal == False and e.scope_distance == 1])
     assert 0 == len([e for e in events if e.type == "OPEN_TCP_PORT" and e.data == "127.0.0.77:8888"])
 
+
+    for e in all_events:
```

Comment:
let's not forget to remove this

Reply:
https://github.com/blacklanternsecurity/bbot/pull/1471/commits/07d0d50c9fb96d2063cce99cfd963e9435c51ea6